### ISSUE TITLE
Very sneakily missing translations for WCAG 2.0

### ISSUE THREAD

**Issue by rudiedirkx** [w3c/wai-wcag-em-report-tool#120]
Both 2.0 and 2.1 specs are included: https://github.com/w3c/wai-wcag-em-report-tool/blob/main/src/data/wcag.json and supported in the tool step 1:

![image](https://user-images.githubusercontent.com/168024/208250274-d9648f96-949e-47ef-9743-4b7a30179dd7.png)

but after choosing 2.0, when you look at the success criteria in step 4:

> 1.3.3: Sensory Characteristics Level A

That's 1.3.3 for WCAG 2.1 (id `sensory-characteristics`), not 2.0 (id `content-structure-separation-understanding`). I don't even know what the correct translation would be, because it's not included anywhere in the tool (? I think ?).

The tool pretends to support two versions, but it translates only one. That's VERY confusing, and a huge bug IMO. The tool should include translations for 2.0, or not support the 2.1 spec.

**Comment by rudiedirkx** [comment]
More of the same bug:

Link "Understanding 1.3.3" goes to https://www.w3.org/WAI/WCAG21/Understanding/content-structure-separation-understanding.html which doesn't exist, because that criterion id doesn't exist in 2.1.

Link "How to meet 1.3.3" magically redirects from https://www.w3.org/WAI/WCAG21/quickref/#content-structure-separation-understanding to https://www.w3.org/WAI/WCAG21/quickref/#sensory-characteristics because that's 1.3.3 in 2.1. VERY confusing.

![image](https://user-images.githubusercontent.com/168024/208250570-e600d9b1-932c-4ff8-a5cc-825185e9e000.png)

**Comment by rudiedirkx** [comment]
And more: if you choose WCAG 2.0 AA, you get 38 assertions/criteria (correct), but with the criteria ids of 2.1 (incorrect).

Choosing WCAG 2.0 is very broken.

**Comment by rvantonisse** [comment]
> I don't even know what the correct translation would be, because it's not included anywhere in the tool (? I think ?).

It is just an version 2.0 id versus 2.1 id. Nothing to do with translations so it is correct they are not included.

The tool was previously supposed to open data from previous versions and 2.0 wcag data would be able to be upgraded to 2.1 data. That is why both ids are there.

Wcag Data and translations are extracted from the official wcag recommendations. Found e.g. here wcag 2.0 : https://www.w3.org/TR/WCAG20/. And example translation: https://www.w3.org/Translations/WCAG20-nl/. The wcag data in this tool should reflect that, as far as I know.

**Comment by rudiedirkx** [comment]
But 2.0 translations aren't the same as 2.1 translations, right? If the criteria id are significantly different (`sensory-characteristics` vs `content-structure-separation-understanding` for criterion 1.3.3), the translations/titles must be different too. But they don't even exist in the tool. But you can start a 2.0 flow. With 2.1 ids and titles, but with the number of items of 2.0. Don't tell me that's not broken.

**Comment by rvantonisse** [comment]
The ids Looks "broken" like you described. But the content differences of wcag 2.0 and 2.1 is similar ( here and there some editotial changes). The translations are correct.

Why criterion titles dont match with ids in 2.0 I cant say / dont know. Translations are based on latest wcag version, because of its backward compatibility. Its mere editorial changes, not changing the meaning of criteria.

Maybe that is why there is only a reference to the latest wcag. It is the most recent updated version criterion text.

When testing specificly for wcag 2.0, a 2.1 version id in the data is indeed confusing.

**Comment by rudiedirkx** [comment]
> Translations are based on latest wcag version, because of its backward compatibility.

Wait whaaat? Is 2.1 a pure extension of 2.0? So 1.3.3 in 2.0 is the exact same 1.3.3 in 2.1, with the same meaning? But but but the ids changed.... That's not very backward compatible. Man, I understand less and less of WCAG every day =)

And what about 2.2, do you know? Will that be a pure extension of 2.1, where 1.3.3 is the same 1.3.3 with the same meaning etc? That's what https://www.w3.org/TR/WCAG22/ sounds like, but I had expected the same from 2.0 to 2.1.

Either way, however WCAG works, I think the report tool is broken for 2.0. I have no idea why anyone would make a 2.0 report, so I would just disable it on https://www.w3.org/WAI/eval/report-tool/evaluation/define-scope
